.github/release-please-config.json

```
@@ -1,6 +1,11 @@
 {
   "separate-pull-requests": true,
   "bootstrap-sha": "c918ffc59eafa01fbc63d5df11ba621cc1888c64",
+  "changelog-sections": [
+    { "type":"feat", "section":"Features", "hidden":false },
+    { "type":"fix", "section":"Fixes", "hidden":false },
+    { "type":"chore", "section":"Other Changes", "hidden":false }
```

Go all in on nice formats?:

```suggestion
    { "type": "feat", "section": "Features", "hidden": false },
    { "type": "fix", "section": "Fixes", "hidden": false },
    { "type": "chore", "section": "Other Changes", "hidden": false }
```